README.md

```diff
@@ -72,14 +76,10 @@ inside your session. After this, if you want quit the Postgres client, simply ty
 
 --- 
 
-With the development database created and python requirements installed we can migrate the tables. We will use the `orator_development.py` file to migrate, run the following command:
+With the development database created and python requirements installed we can migrate the tables. We will use the `orator_config.py` file to migrate, run the following command:
 
 ```bash
-$ orator migrate -c orator_development.py
+$ orator migrate -c orator_config.py
 ```
 
-If you want to recreate a fresh database. Just delete all the tables from the database and repeat the steps above. 
-
-### .env file
-
-Using the `.template.env` fill the missing keys and save it as a **new** copy with the name `.env` at the root directory of the project. Pipenv will use this file to load the environment variables
+If you want to recreate a fresh database. Just delete all the tables from the database and repeat the steps above.
```

Comment:
To recreate the DB one should `DROP DATABASE neo-data` and then create it again, right?

Reply:
@lukasab

Reply:
You're right! I will fix this.